e2e/avpFixtures.js

```diff
@@ -94,7 +94,13 @@ async function scanForA11yViolations(page, testCaseName, options = {}) {
   }
 };
 
-// Function to get JSHeapUsedSize
+/**
+ * Gets the used JS heap size from the page.
+ * 
+ * @param {import('@playwright/test').Page} page - The page from which to get the heap size.
+ * @returns {Promise<number|null>} The used JS heap size in bytes, or null if not available.
+ * @note Can only be executed when running the 'chrome-memory' Playwright config at e2e/playwright-performance-prod.config.js.
+ */
```

Comment:
@unlikelyzero We added a new `SnapshotResultReader` API in MemLab, this eliminates the need to generate the meta file I mentioned early.

```javascript
const reader = SnapshotResultReader.fromSnapshots(
    pathToBaselineSnapshot,
    pathToTargetSnapshot,
    PathToFinalSnapshot,
);
const leaks = await findLeaks(reader);
```

Here is an example test:
https://github.com/facebook/memlab/blob/main/packages/api/src/__tests__/API/E2EResultReader.test.ts#L113-L118